### PR TITLE
Enable linter for common lib

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: Gruntfile.js [hunk 1]
```diff
@@ -67,6 +67,17 @@ module.exports = function(grunt) {
 			}
 		},
 
+		tslint: {
+            build: {
```
Comment (rosen-vladimirov) [inline Gruntfile.js]: mixed tabs and spaces

**Comment by rosen-vladimirov** [comment]
:+1:  after replacing spaces with tabs

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**